sympy/stats/tests/test_continuous_rv.py

```diff
@@ -1260,8 +1260,8 @@ def test_prefab_sampling():
     size = 5
     for var in variables:
         for i in range(niter):
-            assert sample(var) in var.pspace.domain.set
-            samps = sample(var, size=size)
+            assert next(sample(var))[0] in var.pspace.domain.set
```

Comment:
This will raise an `AssertionError` , which the user won't understand. It'd be better to have some message as well.